### PR TITLE
Check in enhanced interval, min max gap, and corner radius samples

### PR DESCRIPTION
**Pull request by agoldenbaum**

### CODE REVIEW

File: samples/charts/data-chart/axis-annotations-corner-radius.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+{
+  "strings": {
+    "legendTitle": "Renewable Electricity Generated"
+  },
+  "descriptions": {
+    "content": {
+      "type": "DataChart",
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/axis-annotations-corner-radius.json:7]: this feature should be showcased on CategoryChart
ask Graham to add these props to CategoryChart if they are missing

File: samples/charts/data-chart/axis-enhanced-intervals.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
+{
+  "export": false,
+  "descriptions": {
+    "content": {
+      "type": "DataChart",
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/axis-enhanced-intervals.json:5]: this feature should be showcased on CategoryChart
ask Graham to add these props to CategoryChart if they are missing

File: samples/charts/data-chart/axis-min-max-gap.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
+{  
+  "strings": {
+    "legendTitle": "Average Temperature Range in New York"
+  },  
+  "descriptions": {
+    "content": {
+      "type": "DataChart",
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/axis-min-max-gap.json:7]: this feature should be showcased on CategoryChart
ask Graham to add these props to CategoryChart if they are missing

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: samples/charts/data-chart/axis-enhanced-intervals.json [hunk 4]
```diff
@@ -0,0 +1,72 @@
+{
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/axis-enhanced-intervals.json:1]: rename file to "axis-label-management.json"

File: samples/charts/data-chart/axis-enhanced-intervals.json [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  "export": false,
+  "descriptions": {
+    "content": {
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/axis-enhanced-intervals.json:4]: add property editors to toggle axis "enhanced*" properties and other axis props listed in this [topic](https://github.com/IgniteUI/igniteui-xplat-docs/blob/vnext/doc/en/components/charts/features/chart-axis-options.md#axis-labels-management)

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

**Comment by agoldenbaum** [comment]
EnhancedIntervalSample
LabelRotationSample

File: samples/charts/data-chart/axis-label-rotation.json [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+      "descriptionType": "DataChart",
+      "isHorizontal": true,
+      "isWrappingEnabled": true,
+      "properties": [
+        {
+            "type": "PropertyEditorPropertyDescription",
+            "propertyPath": "shouldConsiderAutoRotationForInitialLabels",
+            "label": "Rotate Labels?",
+            "shouldOverrideDefaultEditor": true            
+        }
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/axis-label-rotation.json:16]: add 2nd property editor to change autoMarginAndAngleUpdateMode property

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**